lib/default_host.rb

```diff
@@ -1,8 +1,7 @@
 class DefaultHost
   def call(request)
     host = request.host
-    return host if host.end_with?(default_host)
-
+    return host if host.end_with?(default_host) || host == 'test.host'
```

Comment:
I wonder if this is needed because of the `DOMAIN_NAME` set in `.env.test` to `example.com` so that the default for rspec which is `test.host` then conflicts. Another approach might be to try setting `DOMAIN_NAME` to `test.host` for all test envs, in `.env.test` but also in the circleci config. That way we might not need this extra `||` case which I'd rather not have so this is less brittle in the future.

Reply:
Tried this just now, it results in the very error we're trying to fix:

`  ActionController::Redirecting::UnsafeRedirectError:
        Unsafe redirect to "http://test.host/...", pass allow_other_host: true to redirect anyway.
`

I agree that `|| host == 'test.host` is kind of hacky though

Reply:
setting DOMAIN_NAME to nothing results in the redirecting behavior that we want but breaks the tests for default_host

Reply:
I'm wondering if DefaultHost is even needed, we took out subdomain. Maybe try removing the method `default_url_options` from application_controller.rb, I think everything will work how we want it to, and it might resolve this issue, we could then also nuke the test for DefaultHost itself

Reply:
getting rid of default host worked, pushing that commit now